lib/download.ps1

```diff
@@ -138,7 +138,7 @@ function Invoke-Download ($url, $to, $cookies, $progress) {
         # Handle manual file rename
         if ($url -like '*#/*') {
             $null, $postfix = $url -split '#/'
-            $newUrl = "$newUrl#/$postfix"
+            $newUrl = "$newUrl`#/$postfix"
```

Comment:
Why add this `?

Reply:
It's done to escape the `#` symbol. The output string is still the in two cases but by escaping the `#` symbol, the [powershell tree sitter module](https://github.com/airbus-cert/tree-sitter-powershell) can properly recognize the syntax and highlight it correctly in the text editor

Reply:
Okay, so just for tree sitter module? I think PowerShell itself could handle this...

Reply:
Yes, purely for code highlighting. PowerShell handles both case the same